install-dev/theme/views/layout.php

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" >
+  <head>
+    <title><?php echo $this->translator->trans('PrestaShop Installation Assistant', [], 'Install'); ?></title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta http-equiv="Cache-Control" content="no-cache, must-revalidate" />
+    <meta http-equiv="Pragma" content="no-cache" />
+    <meta http-equiv="Cache" content="no store" />
+    <meta http-equiv="Expires" content="-1" />
+    <meta name="robots" content="noindex" />
+    <link rel="shortcut icon" href="theme/img/favicon.ico" />
+    <link rel="stylesheet" type="text/css" media="all" href="theme/view.css" />
+
+    <?php if ($this->language->getLanguage()->isRtl() == 'true') { ?>
+      <link rel="stylesheet" type="text/css" media="all" href="theme/rtl.css" />
+    <?php } ?>
+
+    <script type="text/javascript" src="../js/jquery/jquery-3.5.1.min.js"></script>
+    <script type="text/javascript" src="../js/jquery/jquery-migrate-3.1.0.min.js"></script>
+    <script type="text/javascript" src="../js/jquery/plugins/jquery.chosen.js"></script>
+    <script type="text/javascript" src="theme/js/install.js"></script>
+    <script type="text/javascript" src="https://www.prestashop.com/js/user-assistance.js"></script>
```

Comment:
Remain a PrestaShop call.

Reply:
I agree with @Progi1984 but you marked this conversation as resolved, can we know why? 😄

Reply:
My bad, I thought I fixed it, but forget to push this line :sweat_smile: